src/transforms/deployment.ts

```diff
@@ -7,25 +7,31 @@ import { LoggerWithTarget } from "probot/lib/wrap-logger";
 import { Octokit } from "@octokit/rest";
 import { booleanFlag, BooleanFlags } from "../config/feature-flags";
 import { compareCommitsBetweenBaseAndHeadBranches } from "./util/githubApiRequests";
+import GitHubClient from "../github/client/github-client";
+import {AxiosResponse} from "axios";
 
 // https://docs.github.com/en/rest/reference/repos#list-deployments
 async function getLastSuccessfulDeployCommitSha(
 	owner: string,
 	repoName: string,
 	github: GitHubAPI,
+	newGitHubClient: GitHubClient,
+	useNewClient: boolean,
```

Comment:
Using boolean method parameters is an anti-pattern. 

However now I am doing it to minimise the amount of changes for the PR. Will be gone anyway with the feature flag cleanup.